tests/unit/unit_tests_common.c

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * -----------------------------------------------------------------------------
+ * unit_tests_common.c
+ *
+ * Common functions shared across unit test sources.
+ * -----------------------------------------------------------------------------
+ */
+#include "ctest.h" // This is required for all test-case files.
+#include "unit_tests.h"
+
+/*
+ * Setup function to manage output log streams from most (unit) tests.
+ * By default, there will be no output. Info / error messages are only printed
+ * when the caller sets the VERBOSE env var to opt-in.
+ */
+void
+set_log_streams_for_tests()
+{
+   if (Ctest_verbose) {
+      platform_set_log_streams(stdout, stderr);
+   }
+}
+
+/*
+ * Setup function is provided to manage output log streams from (unit) tests
+ * that are negative tests, checking error conditions / messages.
+ *
+ * Some unit tests exercise error cases, so even when everything succeeds, they
+ * generate lots of "error" messages. By default, that would go to stderr, which
+ * would pollute test output. This function sets up output file handles such
+ * that those expected error messages are only printed when the caller sets the
+ * VERBOSE env var to opt-in. By default, execution from unit-tests will be
+ * silent, redirecting output handles to /dev/null.
+ *
+ * Returns: Output file handles (out/error) if caller has requested for them.
+ */
+void
+set_log_streams_for_neg_tests(platform_log_handle **log_stdout,
```

Comment:
I recognize this function may be difficult to name well, but I found "neg" confusing and inconsistent with our general style of using full descriptive names for functions.  Perhaps instead
```
set_log_streams_for_error_reporting_tests
```
or
```
set_log_streams_for_error_case_tests
```
or
```
set_log_streams_for_expected_error_tests
```

Since this function is always called unconditionally, on its own line, with short arguments, I think the longer name is fine.

Reply:
I settled on a slightly diff version of your 3rd option, i.e.,  `set_log_streams_for_error_tests()`. 

Confirmed off-line w/ you on slack.